.github/workflows/ci.yml

```diff
@@ -3,12 +3,18 @@ name: CI
 on:
   workflow_dispatch:
   pull_request:
-  push:
```

Comment:
I want CI to run on branches that aren't based on `main`.

If we only run CI on branches that are going to merge to `main`, then PR series become a lot harder to test. (Because each PR is based on the previous PR, not `main`.)

Reply:
Maybe we want coverage to run on PR series as well. But that's less important - we can fix it later if you want.

Reply:
Solved here https://github.com/ZcashFoundation/zebra/commit/ed4f4ce45132908b2da2a3bf2aa3dd55e83e9146